react/components/Dialog/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component, Fragment, ReactNode } from 'react'
+import { createPortal } from 'react-dom'
+import OutsideClickHandler from 'react-outside-click-handler'
+
+interface DialogProps {
+  onClose: () => void
+  children: ReactNode
+}
+
+class Dialog extends Component<DialogProps, {}> {
```

Comment:
question, couldn't this be replaced by styleguide's `ModalDialog`?

Reply:
Because the `ModalDialog` is too large in the mobile mode.